src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/SingleSecurityRatingCalculator.java

```diff
@@ -49,11 +54,20 @@
   public SingleSecurityRatingCalculator calculateFor(GitHubProject project) throws IOException {
```

Comment:
Would this function throw an exception if there is a try catch block? Is this `throws IOException` required?

Reply:
Yes, the method can still throw an `IOException`.